samples/Directory.Packages.props

```diff
@@ -11,7 +11,7 @@
 		you need to make sure that the version provided here matches https://github.com/novotnyllc/MSBuildSdkExtras/blob/main/Source/MSBuild.Sdk.Extras/DefaultItems/ImplicitPackages.targets#L11.
 		This is not an issue when libraries are referenced through nuget packages. See https://github.com/unoplatform/uno/issues/446 for more details.
 		-->
-		<PackageVersion Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.11" />
+		<PackageVersion Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.12" />
```

Comment:
Is this needed for something?

Reply:
Sorry I did not see your comment before.
The sample is using 6.2.11 but the rest of the project is at 6.2.12
![image](https://github.com/unoplatform/uno.toolkit.ui/assets/7770915/3319e51e-ad49-4873-99ba-c2af6d2e4410)

Same for Microsoft.Net.Compilers.Toolset, the sample is below the rest of the project.
If it's something we don't need I can close this PR.